commands/Help.js

```diff
@@ -0,0 +1,45 @@
+import Command from "./base/Command.js";
+import { yuuto } from "../yuuto.js";
+
+export default class Help extends Command {
+  constructor() {
+    super({
+      name: "help",
+      category: "info",
+      description: "Get the usage of any other command",
+      aliases: ["usage"],
+      usage: `Run \`help <command>\` to get usage instructions on \`<command>\`, if it exists. Run \`help list\` to list possible commands.`
+    });
+  }
+
+  async run(client, message, args) {
+    const commandName = args[0] || "help";
+
+    if (commandName === "list") {
+      message.channel.send(
+        `Here is a list of all commands and their descriptions:\n${yuuto.commands
+          .map(command => {
+            return ` - **${command.category}:** \`${command.name}\`: *${command.description}.*`;
+          })
+          .join("\n")}`
+      );
+      return;
+    }
+
+    const command =
+      yuuto.commands.get(commandName) ||
+      yuuto.commands.get(yuuto.aliases.get(commandName));
```

Comment:
You already have access to the client here.
```suggestion
      client.commands.get(commandName) ||
      client.commands.get(client.aliases.get(commandName));
```